expyfun/stimuli/_tracker.py

```diff
@@ -239,6 +240,20 @@ def respond(self, correct):
                 if self._direction <= 0:
                     self._direction = 1
 
+        if self._repeat_limit == 'reversals':
+            if self._x[-1] == self._x_min:
+                step_dir = 1
+                self._n_up = 0
+                reversal = True
+                self._n_reversals += 1
+                self._direction = 1
+            if self._x[-1] == self._x_max:
+                step_dir = -1
+                self._n_up = 0
+                reversal = True
+                self._n_reversals += 1
+                self._direction = 1
+
```

Comment:
hmm, I like that idea. I feel like it should be an either or (i.e. warn or property), so for ``check_valid`` I'd want it to say either ``True`` for "yay all good" or some ``int`` to tell how many reversals were ceiling/floor which would replace the functionality of the warning.